CHANGELOG.md

```diff
@@ -7,13 +7,39 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [2.7.0] - November, 22, 2022
+## [2.8.0] - November, 30, 2022
 
 ### Design kit
 
 #### Added
 
 - [Icons] New icon UI/Navigation/swap-user
```

Comment:
- I think this can just be "[Icons] New icon SwapUser" ?
- Also this can be put under React and Core Added sections as well. A good reference example is version [2.5.0](https://github.com/City-of-Helsinki/helsinki-design-system/releases/tag/v2.5.0).